res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1624,15 +1624,18 @@ NumarkMixtrack3.SamplerButton = function(channel, control, value, status, group)
                 engine.setValue(group, "eject", 1);
                 engine.setValue(group, "eject", 0);
             } else {
-                // play sampler with Sync
                 engine.setValue(group, "cue_gotoandplay", 1);
                 engine.setValue(group, "beatsync", 1);
```

Comment:
No, I just thought the code is pretty clear in this case. Shift + Sampler will play the sample without sync so it's pretty easy to switch between the two.

Earlier I thought there'd be an issue with mapping Shift + Sampler to stop a sample, but there isn't one since stopping is only done on playing samples and playing without sync is only done on non-playing samples.